index.html

```diff
@@ -3662,40 +3723,6 @@ <h4>Global Data Checks</h4>
 								</details>
 							</li>
 
-							<li id="global-recurse">
-								<p>Recursively check <var>value</var> as follows to ensure that all properties are
-									verified:</p>
-								<ol>
-									<li id="global-recurse-maps">
-										<p>If <var>value</var> is a <a href="https://infra.spec.whatwg.org/#ordered-map"
-												>map</a>, <a href="https://infra.spec.whatwg.org/#map-iterate">for
-												each</a>
-											<var>key</var> → <var>keyValue</var> of <var>value</var>, set
-												<var>value[key]</var> to the result of running <a>global data checks</a>
-											given <var>key</var> and <var>keyValue</var>.</p>
-									</li>
-
-									<li id="global-recurse-lists">
-										<p>If <var>value</var> is a <a href="https://infra.spec.whatwg.org/#list"
-												>list</a>, <a href="https://infra.spec.whatwg.org/#list-iterate">for
-												each</a>
-											<var>item</var> of <var>value</var>, if <var>item</var> is a <a
-												href="https://infra.spec.whatwg.org/#ordered-map">map</a>, <a
-												href="https://infra.spec.whatwg.org/#map-iterate">for each</a>
-											<var>key</var> → <var>keyValue</var> of <var>value</var>, set
-												<var>value[key]</var> to the result of running <a>global data checks</a>
-											given <var>key</var> and <var>keyValue</var>.</p>
-									</li>
-								</ol>
-								<details>
-									<summary>Explanation</summary>
-									<p>In order to ensure that all the properties in the manifest get processed, this
-										step recursively checks each entry for additional map entries to process. If the
-										value is a list, each item is inspected to determine if it is a map that can be
-										processed.</p>
-								</details>
-							</li>
-
 							<li id="global-end">
 								<p>Return <var>value</var>.</p>
 							</li>
```

Comment:
The explanation at the end of the global data check is empty...

Reply:
Oh, ya, I was initially going to try and write generic explanations for the functions, but it seemed like it would be too late to have them at the end so I put the explanation where the function is called and annotated each step instead. Forgot to take that one back out.